installing/installing-mirroring-installation-images.adoc

```diff
@@ -65,6 +65,37 @@ In production environments, add the required images to a registry in your restri
  modules/installation-performing-disconnected-mirror-without-registry.adoc[leveloffset=+2]
 ////
 
+[id="mirror-registry"]
+== Mirror registry for {product-title}
+
+This section introduces how to install mirror registry for {product-title}, a small and streamlined container registry to use as a target for mirroring the required container images of {product-title} for disconnected installations.
+
+.Prerequisites
+
+* An {product-title} subscription.
+* {op-system-base-full} 8 or Fedora CoreOS with Podman 3.3 installed.
```

Comment:
Should we list Fedora CoreOS? We don't support this OS, in our standard support channels, so is this something we want to 'support' or 'invite' support on. 

cc: @dmesser